java/openmldb-batch/src/main/scala/com/_4paradigm/openmldb/batch/utils/HybridseUtil.scala

```diff
@@ -293,22 +292,21 @@ object HybridseUtil {
     val reader = spark.read.options(options)
 
     val (oriSchema, readSchema, tsCols) = HybridseUtil.extractOriginAndReadSchema(columns)
-    var df = spark.emptyDataFrame
-    if (format != "csv") {
+    var df = if (format != "csv") {
       // When reading Parquet files, all columns are automatically converted to be nullable for compatibility reasons.
       // ref https://spark.apache.org/docs/3.2.1/sql-data-sources-parquet.html
-      df = reader.format(format).load(file)
+      val df = reader.format(format).load(file)
       require(checkSchemaIgnoreNullable(df.schema, oriSchema),
         s"schema mismatch(ignore nullable), loaded ${df.schema}!= table $oriSchema, check $file")
       // reset nullable
-      df = df.sqlContext.createDataFrame(df.rdd, oriSchema)
+      df.sqlContext.createDataFrame(df.rdd, oriSchema)
```

Comment:
test, data null, schema not null, reset nullable ok?

Reply:
reset won't check the data, just modify the schema. The nullable property will affect when we read the real data(e.g. do select).
Note that, the stored data in openmldb will load by autoLoad too, so the schema will be reset.
So the process is: read from hive, hive schema, save to offline storage(parquet), register by autoLoad(reset the schema).
If we reset the schema in hiveLoad, will we get the error when save to offline storage, more early?
online import from hive will report the error if schema not null but the data is null. All we need to do is checking the offline import.